cmd/ssh.go

```diff
@@ -373,15 +295,10 @@ func executeSSH(srcPath, dstPath string, connectionInfo types.ConnectionInfo, pr
 	return nil
 }
 
-// getPrivateKeyPathFromArgs returns the first best private key path from SSH arguments for an Exec
-func getPrivateKeyPathFromArgs(ctx context.Context, e types.Exec, sshArgsByKey map[string]string) string {
+// getUnweavePrivateKeyOrDefault tries to find the private key for this Exec, or relies on a default key if it can't be found
+func getUnweavePrivateKeyOrDefault(ctx context.Context, e types.Exec, defaultKey string) string {
```

Comment:
Function naming can be simpler. We should rely on the doc strings for clarity. Could renmae to `getDefaultKey`